src/components/common/YourCompany/CorrectName/CorrectNameRequest.vue

```diff
@@ -214,33 +214,33 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
             'OK'
           )
         } else {
-          this.parseNameRequest(nr)
+          this.storeNameRequest(nr)
           // Set our entity type, if it's a conversion request
           if (nr.request_action_cd === NrRequestActionCodes.CONVERSION) {
-            this.setEntityType(nr.legalType)
+            this.setEntityType(nr.legalType as unknown as CorpTypeCd)
```

Comment:
This is a weird one and I don't have a fix yet. This is a work-around. Basically, even though `setEntityType` and `nr` use the same import for CorpTypeCd, my IDE says they aren't compatible. 🤷‍♂️